src/reactapp/src/components/shippingMethod/components/ShippingMethodList.jsx

```diff
@@ -47,29 +47,29 @@ function ShippingMethodList({ methodRenderers }) {
           const methodName = `${carrierTitle} (${methodTitle}): `;
           const MethodRenderer = methodRenderers[methodId];
 
-          if (MethodRenderer) {
-            return (
-              <MethodRenderer
-                key={methodId}
-                method={method}
-                selected={selectedMethod}
-                actions={{ change: handleShippingMethodSelection }}
-              />
-            );
-          }
-
           return (
             <li key={methodId} className="flex">
-              <RadioInput
-                value={methodId}
-                label={methodName}
-                name="shippingMethod"
-                checked={selectedMethodId === methodId}
-                onChange={handleShippingMethodSelection}
-              />
-              <span className="pt-2 pl-3 font-semibold">
-                {__('Price: %1', price)}
-              </span>
+              {MethodRenderer ? (
+                <MethodRenderer
+                  key={methodId}
```

Comment:
If you wrap this in an `<li>` element, which has the `key` on it, I don't think you also need to set that on the `MethodRenderer`, unless you use it in the renderer itself. But since you also pass the whole `method` along, it seems like this can be removed.